diff --git a/opera b/opera
index 6c08e2d..f4cf0e6 100755
--- a/opera
+++ b/opera
@@ -41,7 +41,7 @@ done
 [ "$PWD" ] || PWD=`/bin/pwd`
 if [ "$_PERSONALDIR_FROM_CMDLINE_" = NO ]
 then
-  OPERA_PERSONALDIR="$PWD/profile"
+  OPERA_PERSONALDIR="$HOME/.opera-10.50"
   export OPERA_PERSONALDIR
 fi
 
@@ -60,52 +60,23 @@ export OPERA_BINARYDIR
 # Opera enviroment
 if [ -z "$OPERA_DIR" ]
 then
-    OPERA_DIR="$PWD/usr/share/opera"
+    OPERA_DIR="/usr/share/opera"
 fi
 export OPERA_DIR
 
 # Acrobat Reader
 for ACROBATBINDIR in \
-	/usr/local/Adobe/Reader8/bin \
-	/usr/lib64/Adobe/Reader8/bin \
-	/usr/lib/Adobe/Reader8/bin \
-	/usr/X11R6/lib/Adobe/Reader8/bin \
-	/opt/Adobe/Reader8/bin \
-	/usr/Adobe/Reader8/bin \
-	/usr/local/Adobe/Acrobat7.0/bin \
-	/usr/lib64/Acrobat7.0/bin \
-	/usr/lib/Acrobat7.0/bin \
-	/usr/X11R6/lib/Acrobat7.0/bin \
-	/opt/Acrobat7.0/bin \
-	/usr/Acrobat7.0/bin \
-	/usr/local/Acrobat7/bin \
-	/usr/lib64/Acrobat7/bin \
-	/usr/lib/Acrobat7/bin \
-	/usr/X11R6/lib/Acrobat7/bin \
-	/opt/Acrobat7/bin \
-	/usr/Acrobat7/bin \
-	/usr/local/Acrobat5/bin \
-	/usr/lib64/Acrobat5/bin \
-	/usr/lib/Acrobat5/bin \
-	/usr/X11R6/lib/Acrobat5/bin \
-	/opt/Acrobat5/bin \
-	/usr/Acrobat5/bin \
-	/opt/acroread-5.07/bin \
-	/usr/local/Acrobat4/bin \
-	/usr/lib64/Acrobat4/bin \
-	/usr/lib/Acrobat4/bin \
-	/usr/X11R6/lib/Acrobat4/bin \
-	/opt/Acrobat4/bin \
-	/usr/Acrobat4/bin \
+	/opt/Adobe/Reader9/bin \
     ; do
     if [ -d "$ACROBATBINDIR" ]
     then PATH="$PATH:$ACROBATBINDIR"
     fi
 done
 
-# Setting environment relative to current working directory
+
+# Setting environment relative to current working directory <-- no! we will not use current
 # Bundled Qt will be found here unless preloaded by the stance above
-LD_LIBRARY_PATH="$PWD/usr/lib/opera${LD_LIBRARY_PATH:+:}$LD_LIBRARY_PATH"
+LD_LIBRARY_PATH="/usr/lib/opera${LD_LIBRARY_PATH:+:}$LD_LIBRARY_PATH"
 export LD_LIBRARY_PATH
 
 # Run Opera:
